src/wp-includes/blocks/block.php

```diff
@@ -8,6 +8,8 @@
 /**
  * Renders the `core/block` block on server.
  *
+ * @since 6.3.0 Reusable Blocks renamed to Patterns.
+ *
```

Comment:
This file is maintained by Gutenberg and is part of the packages that Core consumes. This is why it does not have an `@since`.

 ```suggestion
```

Reply:
Removed :)